File: Company.py
class Company(object):
    def __init__(self, name, status, market, country, city, funding_value, funding_rounds, funding_per_date):
        self._name = name;
        self._status = status;
        self._market = market;
        self._country = country;
        self._city = city;
        self._funding_value = int(funding_value);
        self._funding_rounds = int(funding_rounds);
        self._funding_per_date = int(funding_per_date)

    def getName(self):
        return self._name;

    def getStatus(self):
        return self._status;

    def getMarket(self):
        return self._market;

    def getCountry(self):
        return self._country;

    def getCity(self):
        return self._city;

    def getFunding_value(self):
        return self._funding_value;

    def getFunding_rounds(self):
        return self._funding_rounds;

    def getFunding_per_date(self):
        return self._funding_per_date;

File: IsYourStartupDyingWeb/app/models/company.rb
class Company < ActiveRecord::Base
  # t.string :name
  # t.string :status
  # t.string :market
  # t.string :country
  # t.string :city
  # t.integer :funding_value
  # t.integer :funding_rounds
  # t.string :first_round_date
  # t.string :last_round_date

  belongs_to :user
  validates :market, presence: true
  validates :country, presence: true
  validates :city, presence: true
  validates :funding_value, presence: true
  validates :funding_rounds, presence: true
  validates :first_round_date, presence: true
  validates :last_round_date, presence: true

  def classify
    self.classify_result = `cd ..; python classify_startup.py \'#{self.name}\' \'#{self.status}\' \'#{self.market}\' \'#{self.country}\' \'#{self.city}\' #{self.funding_value} #{self.funding_rounds} #{self.first_round_date} #{self.last_round_date}`
  end
end
